dotCMS/src/main/java/com/dotcms/auth/providers/saml/v1/DotSamlResource.java

```diff
@@ -301,7 +301,7 @@ public void logout(@PathParam("idpConfigId") final String idpConfigId,
 					Logger.debug(this, () -> "Processing saml logout request for idpConfig id: " + idpConfigId);
 					final String logoutPath = this.samlConfigurationService.getConfigAsString(identityProviderConfiguration,
 							SamlName.DOT_SAML_LOGOUT_SERVICE_ENDPOINT_URL,
-							()->buildBaseUrlFromRequest(httpServletRequest) + "/");
+							()-> "/dotAdmin/#/public/logout");
```

Comment:
do we want to cache bust here?  I think so, e.g.
`"/dotAdmin/?r=" + random + "#/public/logout"`